open_academy/__manifest__.py

```diff
@@ -14,4 +14,13 @@
     "depends": [
         "base"
     ],
+
+    "data": [
+        "security/ir.model.access.csv",
+        "views/course.xml",
```

Comment:
```suggestion
        "views/course_views.xml",
```

I hate this suggestion 🤣, but it's OCA guideline https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#id10